templates/shared/forms/interactive/managed-apps.html

```diff
@@ -3,9 +3,24 @@
     <header class="p-modal__header" style="display: block; border-bottom: 0;">
       <button class="p-modal__close" aria-label="Close active modal" style="margin-left: -1rem">Close</button>
       <h2 class="p-modal__title u-sv1" id="modal-title">Canonical, the managed app experts</h2>
-      <p>No matter where they run, we can manage your applications so you can run your business. Share with us some detail on your project and let's get started.</p>
+      <p>Canonical&rsquo;s app engineers will provide a free quote and architecture assessment &mdash; start by sharing the details below.</p>
     </header>
     <div class="js-pagination js-pagination--1">
+      <div class="js-formfield">
+        <h3 class="p-heading--five">Where do you want to run your apps?</h3>
+        <div class="row u-no-padding u-equal-height">
+          <div class="col-4 u-sv3">
```

Comment:
There's only one col here, looks weird that it wraps when there's so much white space available
```suggestion
          <div class="col-12 u-sv3">
```